src/github_runner_image_builder/templates/cloud-init.sh.j2

```diff
@@ -103,10 +106,37 @@ function install_microk8s() {
         classic_flag="--classic"
     fi
     /usr/bin/snap install microk8s --channel="$channel" $classic_flag
-    /snap/bin/microk8s status --wait-ready
```

Comment:
After installing microk8s, it might be good to wait until ready prior to doing anything.
This was the root cause of a bug we have faced in the past.